### PR TITLE
add new line fix #4

### LINKED ISSUE #4: Second Issue
dua

### PR DESCRIPTION
**Pull request by kadabengarann**
resolve #4

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,6 +10,11 @@
 
 <body>
     <h1>WELCOME TO EARTH</h1>
+
+
+
+    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Earum, nemo! Aperiam placeat ab quasi id itaque optio
```
Comment (jhonny0001) [inline index.html:16]: paan nih?

Comment (kadabengarann) [reply index.html:16]: E

**Review by jhonny0001 — CHANGES_REQUESTED** [review]
tambahe lage

**Review by jhonny0001 — APPROVED** [review]

**Review by jhonny0001 — CHANGES_REQUESTED** [review]
P

**Review by jhonny0001 — CHANGES_REQUESTED** [review]
E

**Review by jhonny0001 — APPROVED** [review]

**Pull request merged**